### PR TITLE
feat: Integrate data from brackend to process and directory views

### PR DESCRIPTION
**Pull request by carlos18bp**
**Description:** Integrate data from brackend to process and directory views

### CODE REVIEW

File: frontend/src/views/process/ProcessList.vue [hunk 1]
```diff
@@ -103,4 +107,15 @@
...
+import { useProcessStore } from "@/stores/process";
+
+const processStore = useProcessStore();
+const processes = ref([]);
+
+onMounted(async () => {
+    await processStore.fetchProcessesData();
+    processes.value = processStore.processes;
+});
+
```
Comment (gustavop-dev) [inline frontend/src/views/process/ProcessList.vue:120]: Delete extra line

**Review by gustavop-dev — COMMENTED** [review]
This PR addresses the frontend updates. However, it’s important to note that we recently discussed changing the `case_type` field in the `Process` model from a `CharField` to a `ChoiceField` for better consistency. This adjustment will impact the frontend as well, so it would be helpful to have it in place from the start. Let me know if we should handle that in this PR or if it will be addressed in a future backend update.

File: frontend/src/views/process/ProcessList.vue (hunk 1 above)
Comment (carlos18bp) [reply frontend/src/views/process/ProcessList.vue:126]: Addressed in [commit](https://github.com/carlos18bp/gym_project/pull/3/commits/7ac2f11757dc85d134b3c413bed38863f729857e)

File: backend/gym_app/management/commands/create_processes.py [hunk 2]
```diff
@@ -20,9 +20,18 @@ def handle(self, *args, **options):
         clients = User.objects.filter(role='client')
         lawyers = User.objects.filter(role='lawyer')
 
-        for _ in range(number_of_processes):
+        # Crear 10 Case
```
[the remark this answers is not in the document]
Comment (carlos18bp) [reply backend/gym_app/management/commands/create_processes.py]: Addressed in [commit](https://github.com/carlos18bp/gym_project/pull/3/commits/f5e5e16529ec0daa1f4dd9bfc50eee685f5e6027)

**Pull request merged**